pmp-client/src/features/changes/commitStoreHelpers.ts

```diff
@@ -0,0 +1,51 @@
+import { Change, ParameterChange, ParameterRevert, Revert } from './types';
+
+export const isParameterRevert = (revert: Revert): revert is ParameterRevert => {
+    return revert.revertType === 'parameter';
+};
+
+export const isRevert = (change: Change): change is Revert => {
+    return (change as Revert).revertType !== undefined;
+};
+
+export const isParameterChange = (change: Change): change is ParameterChange => {
+    return !isRevert(change);
+};
+
+export const compareChanges = (a: Change, b: Change): number => {
+    if (isRevert(a) && isRevert(b)) {
+        return a.commitReference.localeCompare(b.commitReference);
+    }
+
+    if (isParameterChange(a) && isParameterChange(b)) {
+        return compareParameterChanges(a, b);
+    }
+
+    if (isRevert(a)) {
+        return -1;
+    }
+
+    return 1;
```

Comment:
Why are these different?

Reply:
a is revert, b is not => should be ordered one way
b is revert, a is not => should be ordered the other way